src/viam/ur/module/ur_arm.hpp

```diff
@@ -1,7 +1,12 @@
 #pragma once
 
+#include <cmath>
```

Comment:
I don't see that `cmath` is used in this header? If you want it for `ur_arm.cpp` it should be included there.